thorlcr/graph/thgraphmaster.ipp

```diff
@@ -71,43 +71,29 @@ protected:
 
 class graphmaster_decl CThorStatsCollection : public CInterface
 {
+    std::vector<OwnedMalloc<CRuntimeStatisticCollection>> nodeStats;
+    const StatisticsMapping & mapping;
+    CRuntimeSummaryStatisticCollection summary;
```

Comment:
This is more naturally a local variable in getStats()

Reply:
yes, and it's the cause of the x3 anomaly (or how ever many updates there are)